docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -1,6 +1,21 @@
 Initial Repository Setup
 ========================================================
 
+Requirements
+-------------------------------------------
+
+Using Linux is recommended.
+The provided scripts will not work on macOS out of the box, but they may be able to be modified to support it.
+Working under Windows is not recommended.
+
+* dtc (design-tree-compiler)
```

Comment:
I would get rid of the particular requirements (dtc, ...). It should be self-explanatory when it breaks that you need X thing.